### PR TITLE
Migrate ngraph backend test of convolution operations

### PR DESCRIPTION
**Pull request by wilson-seok**
### Details:
 - add convolution in template plugin reference test
 - Check SLT parameters for CPU/GPU for convolution
 - Create visitor API test for convolution
 - Remove backend test for convolution

### Tickets:
 - 64010

### CODE REVIEW

**Comment by wilson-seok** [comment]
@ilyachur @iefode ngraph backend test of convolution operation includes one test which is using op::v1::ConvolutionBackpropData. I think it should move to ConvolutionBackpropData op_reference test. So it is not included in convolution.cpp.

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
Please fix conflicts and code style

**Comment by wilson-seok** [comment]
> Please fix conflicts and code style

@ilyachur Could you help me to know where I can find the location of clang-format violation from my code?
The log from CI doesn't provide that detail information.
image

File: ngraph/test/visitors/op/convolution.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+TEST(attributes, convolution) {
+    NodeBuilder::get_ops().register_factory<op::v1::Convolution>();
+    auto data = make_shared<op::Parameter>(element::f32, Shape{1, 16, 124, 124});
+    auto filters = make_shared<op::Parameter>(element::f32, Shape{2, 16, 3, 3});
+    auto strides = Strides{1, 1};
+    auto pads_begin = CoordinateDiff{1, 2};
+    auto pads_end = CoordinateDiff{1, 2};
+    auto dilations = Strides{1, 1};
+    auto convolution = make_shared<op::v1::Convolution>(data,
+                                                                    filters,
```
Comment (yeonbok) [inline ngraph/test/visitors/op/convolution.cpp:28]: I think this should be algned

Comment (wilson-seok) [reply ngraph/test/visitors/op/convolution.cpp]: Applied it and passed the clang-format! Thanks!

**Comment by ilyachur** [comment]
> > Please fix conflicts and code style
> 
> @ilyachur Could you help me to know where I can find the location of clang-format violation from my code? The log from CI doesn't provide that detail information. image

In general, to fix clang-format you can download a patch from the artifact of code-style job.

**Comment by ilyachur** [comment]
I am not sure that you need to change mkl-dnn

**Comment by wilson-seok** [comment]
> > > Please fix conflicts and code style
> > 
> > 
> > @ilyachur Could you help me to know where I can find the location of clang-format violation from my code? The log from CI doesn't provide that detail information. image
> 
> In general, to fix clang-format you can download a patch from the artifact of code-style job.

I got the point and fixed it. Thanks!

**Pull request closed without merging**